src/schema/v2/searchCriteriaLabel.ts

```diff
@@ -0,0 +1,346 @@
+      .map(Math.round)
+
+    labels.push({
+      name: "Size",
+      value: `h: ${hmin}–${hmax} cm`,
+      field: "height",
+    })
+  }
+
+  return labels
+}
+
+function getWaysToBuyLabels(waysToBuy: {
+  acquireable: boolean
+  atAuction: boolean
+  inquireableOnly: boolean
+  offerable: boolean
+}) {
+  const { acquireable, atAuction, inquireableOnly, offerable } = waysToBuy
+  const labels: SearchCriteriaLabel[] = []
+
+  if (acquireable)
+    labels.push({
+      name: "Ways to Buy",
+      value: "Buy Now",
+      field: "acquireable",
+    })
+
+  if (atAuction)
+    labels.push({
+      name: "Ways to Buy",
+      value: "Bid",
+      field: "atAuction",
+    })
+
+  if (inquireableOnly)
+    labels.push({
+      name: "Ways to Buy",
+      value: "Inquire",
+      field: "inquireableOnly",
+    })
+
+  if (offerable)
+    labels.push({
+      name: "Ways to Buy",
+      value: "Make Offer",
+      field: "offerable",
+    })
+
+  return labels
+}
+
+function getMaterialLabels(materialsTerms: string[]) {
+  if (!materialsTerms?.length) return []
+
+  return materialsTerms.map((term) => {
+    return {
+      name: "Material",
+      value: capitalizeWords(term),
+      field: "materialsTerms",
+    }
+  })
+}
+
+function getLocationLabels(locationCities: string[]): SearchCriteriaLabel[] {
+  if (!locationCities?.length) return []
+
+  return locationCities.map((city) => ({
+    name: "Artwork Location",
+    value: city,
+    field: "locationCities",
+  }))
+}
+
+function getPeriodLabels(majorPeriods: string[]) {
+  if (!majorPeriods?.length) return []
+
+  const DISPLAY_TEXT: Record<string, string> = {
+    "2020": "2020–Today",
+    "2010": "2010–2019",
+    "2000": "2000–2009",
+    "1990": "1990–1999",
+    "1980": "1980–1989",
+    "1970": "1970–1979",
+    "1960": "1960–1969",
+    "1950": "1950–1959",
+    "1940": "1940–1949",
+    "1930": "1930–1939",
+    "1920": "1920–1929",
+    "1910": "1910–1919",
+    "1900": "1900–1909",
+  }
+
+  return majorPeriods.map((period) => {
+    return {
+      name: "Time Period",
+      value: DISPLAY_TEXT[period] ?? period,
+      field: "majorPeriods",
+    }
+  })
+}
+
+function getColorLabels(colors: string[]) {
+  if (!colors?.length) return []
+
+  return colors.map((value) => {
+    const color = COLORS.find((c) => value === c.value)
+    if (!color) throw new Error(`Color not found: ${value}`)
+
+    return {
+      name: "Color",
+      value: color.name,
+      field: "colors",
+    }
+  })
+}
+
+async function getPartnerLabels(partnerIDs: string[], partnerLoader) {
+  if (!partnerIDs?.length) return []
+
+  return Promise.all(
+    partnerIDs.map(async (id) => {
+      const partner = await partnerLoader(id)
+      return {
+        name: "Galleries and Institutions",
+        value: partner.name,
+        field: "partnerIDs",
+      }
+    })
+  )
+}
+
+function capitalizeWords(str: string) {
+  return str.replace(/\b([a-z])/g, (_match, p1) => p1.toUpperCase())
+}
```

Comment:
This was a quick local solution to the capitalization needs here — since I couldn't seem to find something like this in MP.

But I wonder if it makes sense to actually just import https://github.com/artsy/to-title-case into MP?

Reply:
Replaced in ac2bff95257c98543b4006ee7a652e89819ad7c7